nbgitpuller/handlers.py

```diff
@@ -168,6 +184,17 @@ async def get(self):
         )
         await self.flush()
 
+    def combine_qs(self, targetpath):
```

Comment:
- Can you add a docstring clarifying whats done here
- Name this function a bit more verbosly, using query_string over qs

Reply:
I can add the docstring certainly. Currently the naming matches "_qs" functions from urllib.